media-sound/cadence-9999.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# requires proaudio overlay
+
+EAPI=5
+
+inherit git-r3
+
+DESCRIPTION="Collection of tools useful for audio production"
+HOMEPAGE="http://kxstudio.linuxaudio.org/"
+
+SRC_URI=""
+EGIT_BRANCH="master"
+EGIT_REPO_URI="https://github.com/falkTX/Cadence.git"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="+a2jmidid +capture pulseaudio +ladish"
+
+DEPEND="=media-sound/jack-audio-connection-kit-1*
```

Comment:
This isn't upstream under this package. Can this use media-sound/jack2? And is this explicitly incompatible with JACK1?